Description: qteesupplicant.service: make it conditional to /dev/tee0
 Remove bindsto and after dev-tee0.device and replace with a conditional
 in order for systemd to not force a wait timeout on devices without the
 tee device node.
Author: Ricardo Salveti <ricardo.salveti@oss.qualcomm.com>
Origin: upstream, https://github.com/qualcomm/minkipc/pull/52
Last-Update: 2026-05-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: pkg-minkipc/qtee_supplicant/qteesupplicant.service.in
===================================================================
--- pkg-minkipc.orig/qtee_supplicant/qteesupplicant.service.in
+++ pkg-minkipc/qtee_supplicant/qteesupplicant.service.in
@@ -2,8 +2,7 @@
 # SPDX-License-Identifier: BSD-3-Clause-Clear
 [Unit]
 Description=QTEE Supplicant Service
-BindsTo=dev-tee0.device
-After=dev-tee0.device sfsconfig.service
+ConditionPathExists=/dev/tee0
 
 [Service]
 Type=exec
